transport/ws/src/main/java/org/kaazing/gateway/transport/ws/bridge/extensions/idletimeout/IdleTimeoutExtensionFactory.java

```diff
@@ -31,6 +31,12 @@ public String getExtensionName() {
     }
 
     @Override
+    public WebSocketExtension offer(ExtensionHelper extensionHelper, WsResourceAddress address) {
+        // connectors are not sending this extension in handshake request
```

Comment:
4.x (and 5.x) is not sending either. Hence I didn't change in the change set. They use WsCheckAliveFilter which gets added when inactivity.timeout is configured.